book/website/ethical-research/activism/activism-workplace.md

```diff
@@ -0,0 +1,126 @@
+
+> In this template, we assume that our subchapter has three key ideas, your subchapter may have more or less than this. 
+
+> It may be beneficial to run your content through a grammar checker (such as Grammarly) to catch grammatical mistakes.
+-->
+
+<!-- 
+In the label, replace `keyidea1` with a word that best describes the section or key idea you want to explain -->
+(sectioninitials-filename-keyidea1)=
+## Unionisation
+
+> **Talk about your key idea in detail**. Feel free to use images, code blocks, and admonitions to communicate your ideas.
+> You may break the section down into subsections if you wish, however, remember to add [labels](https://the-turing-way.netlify.app/community-handbook/style/style-crossref.html) to any additional headers you create to facilitate cross-referencing.
+
+> Remember to use the [style guide](https://the-turing-way.netlify.app/community-handbook/style.html) and Jupyter Book's [Cheat Sheet](https://jupyterbook.org/reference/cheatsheet.html) to guide your writing.
+> The [style guide](https://the-turing-way.netlify.app/community-handbook/style/style-citing.html) also contains _The Turing Way's_ recommendations for referencing and citation.
+
+> To include an image in your writing, use the MyST directive shown below. 
+> Remember to add your image to the `figures` [folder](https://github.com/alan-turing-institute/the-turing-way/tree/master/book/website/figures) and use the correct path, else it will not be displayed.
+
+```{figure} ../../figures/image-name.png
+---
+name: image-name
+alt: describe your image for readers who rely on screen readers
+---
+Your image caption here
+```
+
+> To include code blocks, simply enclose your code in three sets of backticks shown below.
+
+```
+def simple_function():
+    pass
+```
+
+> To include an admonition or to highlight a block of text that exists slightly apart from the narrative of your section, use the directive shown below. Jupyter Book's [documentation](https://jupyterbook.org/content/content-blocks.html#) has other useful examples.
+
+```{note}
+Here is a note!
+```
+
+<!-- 
+In the label, replace `keyidea2` with a word that best describes the section or key idea you want to explain -->
+(sectioninitials-filename-keyidea2)=
+## Internal policy advocacy
+
+> **Talk about your key idea in detail**. Feel free to use images, code blocks, and admonitions to communicate your ideas.
+> You may break the section down into subsections if you wish.
+> Remember to use the [style guide](https://the-turing-way.netlify.app/community-handbook/style.html) and Jupyter Book's [Cheat Sheet](https://jupyterbook.org/reference/cheatsheet.html) to guide your writing.
+> The [style guide](https://the-turing-way.netlify.app/community-handbook/style/style-citing.html) also contains _The Turing Way's_ recommendations for referencing and citation.
+
```

Comment:
```suggestion
Internal policy advocacy is when individuals or groups associated with an organisation campaign for a change in the way that the organisation makes decisions, runs projects, chooses collaborators, develops products, etc.
It can be a really powerful tool for change, regardless of whether or not the group of people campaigning are in a union.
### Example: Google Workers & Project Maven
One example of this is Google workers campaigning against Project Maven, a contract with the US Pentagon to use AI to develop more effective drone guidance.
Google employees across the globe spoke up in internal meetings, signed petitions, made memes, and shared personal statements about why they disagreed with the contract.
Some employees even resigned.
Eventually, the leadership at Google agreed not to seek further contracts with the US Pentagon.
[Source, accessed 8 December 2020](https://jacobinmag.com/2018/06/google-project-maven-military-tech-workers)
```

Reply:
amazing thanks Arielle! I've committed these into the draft.